packages/product-json-to-xlsx/src/writer.js

```diff
@@ -0,0 +1,144 @@
+/* @flow */
+import fs from 'fs'
+import tmp from 'tmp'
+import path from 'path'
+import slugify from 'slugify'
+import archiver from 'archiver'
+import EmitOnce from 'single-emit'
+import { mapHeaders } from '@commercetools/product-json-to-csv'
+import mapValues from './map-values'
+import ProductExcel from './product-excel'
+
+// Accept a highland stream and write the output to a single file
+export function writeToSingleXlsxFile(
+  productStream: Object,
```

Comment:
Niptick: how difficult is it to have this object typed?

Reply:
Flow type for highland stream added here f10d31d